polaris-react/src/components/TextField/TextField.tsx

```diff
@@ -756,6 +766,15 @@ export function TextField({
     );
   }
 
+  function isTextArea(target: HTMLElement | EventTarget) {
```

Comment:
This seems to be identical to `isInput()`, which then I wonder if there's a need for separating out these checks, especially now given that `getInput` abstracts out which ref is in use (i.e. change `isInput` to use `getInput` instead of `inputRef`).

Reply:
I was thinking it may be better to have it separate if there's ever a future case where `textarea` vs `input` needs to be specifically targeted?

Reply:
Let's optimize for code simplicity and maintenance today where possible; if we run into that case in the future we can refactor 👍